app_zarzadzanie_budzetem/KANTOR/Kantor_przeliczanie_walut.py

```diff
@@ -0,0 +1,85 @@
+import requests
+
+
+class Kantor:
+    #Funkcja wyświetlania walut
+    @staticmethod
+    def convert_currency(amount, from_currency, to_currency):
+
+        if from_currency == to_currency:
+            return amount  # No conversion needed
+
+        url = f"https://api.frankfurter.app/latest"
+        params = {
+            "amount": amount,
+            "from": from_currency,
+            "to": to_currency
+        }
+
+        response = requests.get(url, params=params)
+        if response.status_code != 200:
+            raise Exception(f"Error fetching exchange rates: {response.status_code} {response.text}")
+
+        print(response)
```

Comment:
to chyba tylko do diagnostyki biało być?
```suggestion
        #print(response)
```